packages/keystone/src/types/config/access-control.ts

```diff
@@ -7,102 +7,137 @@ type BaseAccessArgs = {
   context: KeystoneContext;
 };
 
-type CreateAccessArgs<GeneratedListTypes extends BaseGeneratedListTypes> = BaseAccessArgs & {
+// List Filter Access
+
+type FilterOutput<GeneratedListTypes extends BaseGeneratedListTypes> =
+  | boolean
+  | GeneratedListTypes['inputs']['where'];
+
+export type ListFilterAccessControl<
+  Operation extends 'create' | 'query' | 'update' | 'delete',
+  GeneratedListTypes extends BaseGeneratedListTypes
+> =
+  | FilterOutput<GeneratedListTypes>
```

Comment:
It feels inconsistent to allow the value directly here but not for the others. (i _think_ we should require a function here as well)